src/routes/+layout.svelte

```diff
@@ -2,9 +2,11 @@
     import '../global-styles.scss'
 </script>
 
-<nav>
-	cool navbar
-</nav>
+<div class="mobile-sized-container">
```

Comment:
I'd go back to using nav instead of div since it looks like this is becoming our navbar.  Also consider placing the navbar in header if this will become part of the app's repeated header element.

Reply:
I agree with putting the home button into a <nav>.

But the darn `<nav>` makes it have too much margin until I add this

![image](https://github.com/user-attachments/assets/406d5fd3-5cf6-48b1-b8f6-1aa655530642)